backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientSelfRegistrationLink.java

```diff
@@ -2,18 +2,19 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.JoinColumn;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 @Entity
 @Table(name = "patient_registration_link")
 public class PatientSelfRegistrationLink extends EternalAuditedEntity {
-  @OneToOne(optional = true)
+  @OneToOne(optional = true, fetch = FetchType.LAZY)
```

Comment:
**suggestion (non-blocking):** I think these fetch hints don't do anything? Might consider removing them & checking that it has no impact, just to avoid future confusion

Reply:
I just checked and these actually do both work and removing each re-introduces (separate) n+1 issues on the self registration link.  With the data loader, it actually makes it n+2 since the data loader still separately fetches the data.

Agree about the confusion, though.  It is tough to tell when these hints make a difference and when they don't, but I'm not sure how we can improve it.

Reply:
Perfect! Thanks for checking